docs/rfc/active/000-ml_models/README.md

```diff
@@ -0,0 +1,121 @@
+- Start Date: 08/18/2020
+- RFC PR: https://github.com/linkedin/datahub/pull/1812
+- Implementation PR(s): https://github.com/linkedin/datahub/pull/1721
+
+# Machine Learning Models
```

Comment:
Might be useful to add scope for this RFC. For example, this RFC doesn't cover much about model evolution/versions and how we will handle it. I think that might need its own RFC but it will be useful to call out that this design doesn't address model versioning on linking related models together and that we will address it separately.

Reply:
Added it under future work

Reply:
Thanks for the suggestion.